builtin/providers/google/resource_google_project.go

```diff
@@ -210,6 +231,18 @@ func resourceGoogleProjectRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("policy_etag", pol.Etag)
 	d.Set("policy_data", string(polBytes))
 
+	// Read the billing account
+	ba, err := config.clientBilling.Projects.GetBillingInfo("projects/" + pid).Do()
```

Comment:
@evandbrown it's sadder than that 😦 . Steps to repro:

1. Compile Terraform without this PR
2. Create project
3. Compile Terraform with this PR
4. Change a project attr
5. Get error